app/views/core/CreateAccountModal/CreateAccountModal.js

```diff
@@ -294,6 +297,10 @@ module.exports = (CreateAccountModal = (function () {
 
     afterRender () {
       super.afterRender()
+
+      if (this.signupState.get('path') === 'individual' && features?.china && this.signupState.get('screen') === 'segment-check') {
+        this.signupState.set({ birthday: '2005-01-01', screen: 'basic-info' })
```

Comment:
do we need to set birthday? Is it a required field?

Reply:
i'm not sure but seems normal register ask to set birthday so I set a default one to avoid any issue

Reply:
It doesn't seem to be required based on schema, can you remove it? I think it might screw up analytics if we used this metric to figure out the age of users 😅

Reply:
okay. let me test if there's no birthday